bin/start-server.js

```diff
@@ -1,10 +1,17 @@
 'use strict'
 
+const mongoose = require('mongoose')
+
 const server = require('../lib')
+const dbConfig = require('../../db-config')
 
 const cliPort = process.argv.length > 2 && process.argv[2] || undefined
 const port = cliPort || process.env.PORT || 3000
 
+mongoose.Promise = global.Promise
+const db = process.env.NODE_ENV === 'production' ? dbConfig.prod : dbConfig.dev
```

Comment:
wird diese Zeile auch mit gesetztem 'production' getestet? sonst muss man aufpassen, dass das bei coveralls nicht durchrutscht, bloß weil es in einer Zeile steht haha
Wäre allerdings auch halb so wild